controllers/discordactions.js

```diff
@@ -72,19 +72,11 @@ const createGroupRole = async (req, res) => {
 const getAllGroupRoles = async (req, res) => {
   try {
     const { groups } = await discordRolesModel.getAllGroupRoles();
-    const dev = req.query.dev === "true";
-    if (dev) {
-      // Placing the new changes under the feature flag.
-      const discordId = req.userData?.discordId;
-      const groupsWithMembershipInfo = await discordRolesModel.enrichGroupDataWithMembershipInfo(discordId, groups);
-      return res.json({
-        message: "Roles fetched successfully!",
```

Comment:
why you removed this message

Reply:
its not removed their were two return statement for if and else. i just removed duplicate for if else .